multi/imxrt-multi/config.h

```diff
@@ -766,6 +766,63 @@
 #endif
 #endif
 
+#if !ISEMPTY(UART_CONSOLE)
+#if UART_CONSOLE <= 0
+#error "Invalid value for UART_CONSOLE"
+#endif
+#endif
+
+#if defined(UART_CONSOLE_USER)
+#if !ISEMPTY(UART_CONSOLE_USER)
+#if (UART_CONSOLE_USER <= 0)
+#error "Invalid value for UART_CONSOLE_USER"
+#endif
+#endif
+#else
+#define UART_CONSOLE_USER UART_CONSOLE
+#endif
+
+
+/* RTT */
+
+#ifndef RTT_CHANNEL0
```

Comment:
Maybe let's use the current convention for now, i.e. `RTTx`, `RTTx_BLOCKING`, even though channels are 0-based. Users are familiar with it

Reply:
Done

Reply:
Is there a reason for using `RTT_0`, `RTT_0_BLOCKING` etc. notation instead of `RTT0` (as described previously: `RTTx`, `RTTx_BLOCKING`)? UARTs, SPIs, and other devices in config.h/board_config.h use this notation (e.g. `UART1`, `SPI2_SCK`)

Reply:
I overlooked this, now it's fixed.